diff --git a/dist/layouts/docs/sidebar.js b/dist/layouts/docs/sidebar.js
index ab856d6e6a1e17f6c26fcf9b77c8e106761214c3..983a27c6a60ff3b24aba3e92a7548d52aa1c91b3 100644
--- a/dist/layouts/docs/sidebar.js
+++ b/dist/layouts/docs/sidebar.js
@@ -85,7 +85,7 @@ export function SidebarItem({ icon, ...props }) {
     const pathname = usePathname();
     const active = props.href !== undefined && isActive(props.href, pathname, false);
     const { prefetch } = useInternalContext();
-    return (_jsxs(Link, { ...props, "data-active": active, className: cn(itemVariants({ active })), prefetch: prefetch, children: [icon ?? (props.external ? _jsx(ExternalLink, {}) : null), props.children] }));
+    return (_jsxs(Link, { ...props, "data-active": active, className: cn(itemVariants({ active }), props.className), prefetch: prefetch, children: [icon ?? (props.external ? _jsx(ExternalLink, {}) : null), props.children] }));
 }
 export function SidebarFolder({ defaultOpen = false, ...props }) {
     const [open, setOpen] = useState(defaultOpen);
